### PR TITLE
gh-94781: Clean previous instances of ouput files on Windows

### PR DESCRIPTION
**Pull request by CharlieZhao95**
The files generated by `PCbuild\build.bat` in the `Python\deepfreeze` and `Python\frozen_modules` directories may cause errors when rebuilding after a checkout. 
Now `PCbuild\clean.bat` can correctly remove those previous output files.

* Issue: gh-94781

### CODE REVIEW

**Comment by zware** [comment]
As far as I can tell, this looks good; nice job!  It should get an approval from someone with fresher MSBuild experience before merge, though.

**Review by zooba — APPROVED** [review]

**Comment by zooba** [comment]
Yep, this seems to be spot on. Nicely done!

**Pull request merged**
